HymnsUITests/EarlGrey.swift

```diff
@@ -0,0 +1,188 @@
+//
+// Copyright 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import EarlGrey
+import Foundation
+
+public func GREYAssert(_ expression: @autoclosure () -> Bool, reason: String) {
+  GREYAssert(expression(), reason, details: "Expected expression to be true")
+}
```

Comment:
What are these things? Are these part of the set up? Or are these tests? I can't tell

Reply:
EarlGrey.swift is an autogenerated file as part of the setup to give us these functions for the tests. EarlGreyExtensions.swift are also extensions to give us more functionality for use in our tests. Only EarlGreyUITests.swift are the tests that I wrote.